README.md

```diff
@@ -33,14 +40,26 @@ reporting and documentation from a central piece of information.
 You can run a Maven project by using the Maven Docker image directly,
 passing a Maven command to `docker run`:
 
+### Linux
+
     docker run -it --rm --name my-maven-project -v "$(pwd)":/usr/src/mymaven -w /usr/src/mymaven maven:3.3-jdk-8 mvn clean install
 
+### Windows
+
+    docker run -it --rm --name my-maven-project -v "$(Get-Location)":C:/Src -w C:/Src maven:3.3-jdk-8-windows mvn clean install
```

Comment:
Is this Powershell? If so, maybe

````suggestion
```powershell
docker run -it --rm --name my-maven-project -v "$(Get-Location)":C:/Src -w C:/Src maven:3.3-jdk-8-windows mvn clean install
```
````

AFAIK the batch equivalent would be

```bat
docker run -it --rm --name my-maven-project -v %CD%:C:/Src -w C:/Src maven:3.3-jdk-8-windows mvn clean install
```